.actions-docker-compose.yml

```diff
@@ -1,19 +1,20 @@
 version: "2.4"
+name: newsroom-e2e-tests
 services:
     redis:
         image: redis:alpine
         ports:
             - "6379:6379"
 
     mongo:
-        image: mongo:3.6
+        image: mongo:4
```

Comment:
Would be good to also update this to MongoDB 6 (see superdesk-core's [docker](https://github.com/superdesk/superdesk-core/blob/async/docker-compose.yml))

Reply:
done